--- origsrc/getdata-0.7.3/bindings/python/Makefile.am	2011-03-14 16:54:05.000000000 -0500
+++ src/getdata-0.7.3/bindings/python/Makefile.am	2011-10-18 17:42:41.411412400 -0500
@@ -34,7 +34,7 @@ python_LTLIBRARIES = pygetdata.la
 
 BUILT_SOURCES = pyconstants.c
 AM_CFLAGS = ${WALL} -fno-strict-aliasing
-AM_CPPFLAGS = $(CPPFLAGS) $(PYTHON_CPPFLAGS) $(NUMPY_CPPFLAGS)
+AM_CPPFLAGS = $(CPPFLAGS) $(PYTHON_CPPFLAGS) $(NUMPY_CPPFLAGS) -I$(top_srcdir)/src
 pygetdata_la_LDFLAGS = -module -avoid-version \
 											 -export-symbols-regex initpygetdata
 pygetdata_la_LIBADD = $(PYTHON_LIBS) ../../src/libgetdata.la
